app/controllers/epp/sessions_controller.rb

```diff
@@ -13,6 +13,9 @@ def login
       webclient_request = ENV['webclient_ips'].split(',').map(&:strip).include?(request.ip)
       if webclient_request && !Rails.env.test? && !Rails.env.development?
         client_md5 = Certificate.parse_md_from_string(request.env['HTTP_SSL_CLIENT_CERT'])
+        if ENV['cert_path'].nil?
```

Comment:
- `present?` is better here, since it also handles empty string, whereas `nil?` doesn't.

Reply:
styleguide seems to prefer blank